pom.xml

```diff
@@ -156,7 +156,7 @@ flexible messaging model and an intuitive client API.</description>
     <hdfs-offload-version3>3.3.1</hdfs-offload-version3>
     <json-smart.version>2.4.7</json-smart.version>
     <opensearch.version>1.2.4</opensearch.version>
-    <presto.version>332</presto.version>
+    <presto.version>334</presto.version>
```

Comment:
It's somewhat unrelated to this PR, but these are some very old versions of Presto. 334 was released in May 2020. https://trino.io/docs/current/release/release-334.html

Reply:
The reason to upgrade to 334 is to do the minimal change to fix the issue. There might be incompatibilities in newer versions which would have to be addressed. 
Yes, we should switch to Trino, but it seems that the PR by @MarvinCai https://github.com/trinodb/trino/pull/8020 hasn't proceeded.

Reply:
Since this drops support for JDK8 the documentation needs to be changed as well.

Reply:
I posted on dev mailing list about this: https://lists.apache.org/thread/3yhz8rql4hwch09728b6myfdq0o37gqj

Reply:
#14988 contains a workaround to support both JDK 8 and JDK 11. That is needed for the maintenance branches. For master branch we are going to lift the requirement for the server side components soon, as discussed in the community meeting.